src/pattern-library/components/patterns/feedback/PopoverPage.tsx

```diff
@@ -127,6 +127,20 @@ export default function PopoverPage() {
               </Library.InfoItem>
             </Library.Info>
           </Library.Example>
+          <Library.Example title="onClose">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                Callback invoked when the popover is automatically closed by
+                clicking away or pressing <code>Escape</code>.
+                <br />
+                You are responsible to use this callback to keep local state in
+                sync, setting <code>open</code> to <code>false</code>.
```

Comment:
```suggestion
                The caller should use this callback to keep local state in
                sync, so that the popover is re-rendered with <code>open</code> set to <code>false</code>.
```